data/build.js

```diff
@@ -74,6 +74,73 @@ function createTreeSorter(name) {
   };
 }
 
+async function buildScientificNameToImageMap(names) {
+  function partition(arr, size) {
+    const numGroups = Math.ceil(arr.length / size);
+    const groups = new Array(numGroups).fill(null);
+
+    return groups.map((_, i) => arr.slice(i * size, i * size + size));
+  }
```

Comment:
Create 2d array with the max size of each subarray being "size". In this case size = 50.